### DISCUSSION TITLE
Customer Account API - Missing variant handle in customer order query

### DISCUSSION THREAD

**Discussion by sbaleno** [Shopify/storefront-api-feedback#249] [Ideas & Requests]
While using the Customer Account Graphql API in a Hydrogen site - I am fetching the customer's orders.  I am able to fetch the order and it's line items, but not the product/variant handle associated with the line item.  I would have to fetch the product and it's handle using the productId.  This is not ideal.

Proposal: Allow the handle to be returned

**Comment by juanpprieto** [comment]
eg https://shopify.dev/docs/api/customer/2024-07/objects/LineItem is missing a way to build a variant's product url because there's no way to get the `handle` to use in `/products/$handle...`
